.github/workflows/ci.yml

```diff
@@ -75,6 +75,12 @@ jobs:
       with:
         dotnet-version: ${{ env.DOTNET_VERSION }}
 
+    - name: Delete old nuget packages
+      uses: actions/delete-package-versions@v5
+      with:
+        package-name: 'Arc.StringSanitizer'
+        package-type: 'nuget'
```

Comment:
This delete step is failing if the package does not exist.

Reply:
Instead of deleting all the versions, I have deleted the specific version which I am pushing.